.evergreen/config/test-tasks.yml

```diff
@@ -0,0 +1,28 @@
+tasks:
+  - name: "test-atlas-connectivity"
+    tags: ["atlas", "nodb"]
+    commands:
+      - func: "decrypt atlas URI list"
+      - func: "run tests"
+        vars:
+          TESTS: "tests/atlas.phpt"
+
+  # Test different storage engines
+  - name: "test-mongodb-4.0-standalone-inmemory"
+    tags: ["standalone", "local", "4.0", "storage-engines"]
+    commands:
+      - func: "bootstrap mongo-orchestration"
+        vars:
+          TOPOLOGY: "server"
+          MONGODB_VERSION: "4.0"
+          STORAGE_ENGINE: "inmemory"
+      - func: "run tests"
+  - name: "test-mongodb-4.0-standalone-mmapv1"
+    tags: ["standalone", "local", "4.0", "storage-engines"]
```

Comment:
I didn't see this in `architecture.md`, but I think it'd be helpful to maintain a glossary of the possible tags we might see within the configs. Offhand, I'm not sure how "local" is used.